app/src/main/java/com/infomaniak/mail/ui/main/thread/AttachmentAdapter.kt

```diff
@@ -41,16 +44,36 @@ class AttachmentAdapter(
         val attachment = attachments[position]
 
         fileName.text = attachment.name
-        // TODO: Find how to add the FileType prefix before the file' size.
-        fileDetails.text = /*item.mimeType + " - " + */FormatterFileSize.formatShortFileSize(context, attachment.size.toLong())
+        fileDetails.text = FormatterFileSize.formatShortFileSize(context, attachment.size.toLong())
         icon.load(attachment.getFileTypeFromExtension().icon)
```

Comment:
You must do as in kDrive, display the file thumbnail directly from the local storage.
https://github.com/Infomaniak/android-kDrive/blob/master/app/src/main/java/com/infomaniak/drive/utils/FileItemUtils.kt#L131